gcp/compute_engine/cloudrun_https_lb_serverless_neg/main.tf

```diff
@@ -0,0 +1,52 @@
+provider google {
+  project = var.cloudrun_lb_projectid
+}
+
+provider google-beta {
+  project = var.cloudrun_lb_projectid
+}
+
+resource google_compute_region_network_endpoint_group serverless_neg {
+  provider              = google-beta
```

Comment:
Does this resource need to use google-beta provider? Seems to be available in the standard google provider. 
Might want to specify provider versions

Reply:
agreed

Reply:
Google-beta provider is not needed. Removed it on the deployment module:
https://bitbucket.org/protraderseco/cryptotraders-platform/src/0f0f6cf92ddfd2d2cba381b4275214cfe9c77d49/deployment/terraform/serverless-https-lb/main.tf?at=feat%2FPS-77%2Fhttps_lb#lines-1